Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt

```diff
@@ -186,7 +186,7 @@ fun EpisodeListItem(
 
         Text(
             text = episode.title,
-            maxLines = 2,
+//            maxLines = 2,
```

Comment:
What about leaving the maxLines, but adding an ellipsis? You can add `overflow = TextOverflow.Ellipsis`. Reason for this is that we don't want an extremely long title to create an extremely long list item, so some max value is needed.

Reply:
I agree with your viewpoint, if the title is too long, then the whole lazycolumn will also be very long. At the same time, make the overflown text as consecutive dots(ellipsis notation) is necessary because it could remind users that this part of text is much longer. If we do not add overflow properties, users could be misled and unaware of the overflown text. I will add the overflow properties and fix maxLines as well. :)

Reply:
Well, It is the first time that I opened a pull request for Google, and I have just signed the `Contributor License Agreements` on `https://cla.developers.google.com/` and sent the request to Google Gropu as well. If it is not enough, please let me know. Thank you! :)